Cement.Net/CementEntry.csproj

```diff
@@ -107,6 +111,7 @@
       <MergeAssemblies Include="$(OutputPath)\SharpYaml.dll" />
       <MergeAssemblies Include="$(SolutionDir)\externals\log4stash.dll" />
       <MergeAssemblies Include="$(OutputPath)\NuGet.exe" />
+      <MergeAssemblies Include="$(OutputPath)\JetBrains.Annotations.dll" />

```

Comment:
Use https://github.com/vostok/devtools/blob/master/jetbrains-annotations/JetBrainsAnnotations.cs without merging some dlls.
Copy-paste, or git submodule.

Reply:
Why, though?
This is inconsistent: we don't copy-paste code from `SharpYaml` or `NDesk.Options`.
`JetBrains.Annotations.dll` is not so different from those dependencies.

Also, copy-pasting external library code is usually a big no-no, I don't see why there should be an exception for this case.